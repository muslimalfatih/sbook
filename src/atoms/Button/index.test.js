import renderer from 'react-test-renderer';
import { Button } from './index'

describe('<Button />', () => {
  it('Matches the snapshot', () => {
    expect(renderer.create('<Button/>').toJSON()).toMatchSnapshot();
  })
})
