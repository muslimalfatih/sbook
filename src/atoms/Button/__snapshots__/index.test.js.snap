// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Matches the snapshot 1`] = `"<Button/>"`;
